spec/sessions/log_in_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+require_relative '../helpers/spec_helper'
+require_relative '../helpers/auth_helper'
+require_relative '../helpers/logs_helper'
+require_relative '../../app/controllers/sessions_controller'
+require_relative '../../app/controllers/records_controller'
+require 'yaml'
+
+data = YAML.load_file('data.yml')
+
+RSpec.describe '[Log in API]' do
+  include AuthHelper
+  include LoggerHelper
+
+  def app
+    Rack::Builder.new do
+      run ApplicationController
+      use RecordsController
+      use SessionsController
+      use UsersController
+    end.to_app
+  end
+
+  describe 'GET /log_in_form' do
```

Comment:
could be `context  'when GET /log_in_form' do`

Reply:
adjusted ✅